Duplicati/CommandLine/Commands.cs

```diff
@@ -394,7 +394,7 @@ public static int List(TextWriter outwriter, Action<Duplicati.Library.Main.Contr
                     (res.Files == null || !res.Files.Any()) &&
                     !compareFilter.Empty;
 
-                if (isRequestForFiles && !Library.Utility.Utility.ParseBoolOption(options, "all-versions"))
+                if (isRequestForFiles && !(options.ContainsKey("all-versions")))

```

Comment:
Is it wise the "help" the user by adjusting the all-versions, if there are no matching files?  It seems surprising.  Why not just exit in that case?